lib/elixir/lib/code/normalizer.ex

```diff
@@ -182,6 +182,20 @@ defmodule Code.Normalizer do
     end
   end
 
+  # Tuples
+  defp do_normalize({:{}, meta, args} = quoted, state) do
+    {last_arg, args} = List.pop_at(args, -1)
+
+    with [{{:__block__, key_meta, _}, _} | _] <- last_arg, :keyword <- key_meta[:format] do
+      args = normalize_kw_args(args, state)
+      kw_list = normalize_kw_args(last_arg, state)
+      {:__block__, [], [{:{}, meta, args ++ kw_list}]}
```

Comment:
Do we need the block here? It is not a literal… but I am not sure.

Reply:
Actually, no, I'll remove that. Literal 2-tuples do need a block, as that's how they're encoded by the literal encoder, but 3+ tuples don't need it.